DataAccess/DataAccess.Postgre/PostgreRepositories.cpp

```diff
@@ -188,8 +188,7 @@ Utility::ChannelCreateCodes ChannelsRepository::createChannel(const Models::Chan
     _pTable->Insert()->columns(SubscribNewChannelData)->execute();
 
     Base::Logger::FileLogger::getInstance().log(
-        '['+ "channel id: " + std::to_string(IDNewChannel) +
-        "] Creating channel was successful\n",
+        std::string("Creating channel was successful\n"),
```

Comment:
I think, when we create message for logs, we should give more information about our actions. I suggest leave ID of channel or add the name of new channel in log message

Reply:
Very interesting... OK